faker/exceptions.py

```diff
@@ -0,0 +1,8 @@
+class BaseFakerException(Exception):
+    """The base exception for all Faker exceptions."""
```

Comment:
Surprised this wasn't already a thing, some of the state address providers throw naked Exceptions already...